tests/Zend/Auth/Adapter/DbTable/BasicSqliteTest.php

```diff
@@ -73,6 +73,18 @@ class Zend_Auth_Adapter_DbTable_BasicSqliteTest extends PHPUnit_Framework_TestCa
      */
     public function setUp()
     {
+        if (!defined('TESTS_ZEND_AUTH_ADAPTER_DBTABLE_PDO_SQLITE_ENABLED') ||
```

Comment:
This code (there will be a fair amount of some style of this in other tests) comes from the respective `AllTests.php`. The difference here is that the old ZF1 test suite wouldn't even add certain tests to the suite if these conditions weren't met (which reduces the number of skipped tests encountered, and maybe reduces the amount of memory used, but not enough to be able to run the entire suite at once).